ui/StatusQ/src/submodelproxymodel.cpp

```diff
@@ -5,6 +5,20 @@
 #include <QQmlContext>
 #include <QQmlEngine>
 
+#include <memory>
+
+namespace {
+    const QString roleSuffix = QStringLiteral("Role");
```

Comment:
Could be:
```cpp
constexpr const auto roleSuffix = "Role";
```
(no need to allocate for a `QString`)

Reply:
Ok, fixed. I think there is still some room for improvement (regarding how this literal is used later), but for now it's ok I think. I plan some future changes in this area and will keep in mind to rethink it a bit :)